badge/badge.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+:root {
+    --red: red;
+    --primary-color: #2563EB;
+    --white: white;
+}
+
+.badge-container {
+    max-width: 20rem;
+    margin: auto;
+}
+
+.badge {
+    position: relative;
+    font-size: 0.79rem;
+    border-radius: 2rem;
+    color: var(--white);
+    font-weight: bold;
+    width: max-content;
+    background-color: var(--red);
+}
+
+.text {
+    margin: 2rem;
+}
+
+.badge-on-text {
+    top: -0.8rem;
+    background-color: var(--red);
+    padding: 0.2rem 0.5rem;
+}
+
+.large-badge-on-text {
+    top: -0.8rem;
+    left: -0.5rem;
+    padding: 0.5rem 0.5rem;
+}
+
+.button1 {
+    padding: 0.6rem 0.8rem;
+    background-color: var(--primary-color);
+    color: var(--white);
+    border-radius: 0.5rem;
+    border: 0px;
+}
+
+.badge-on-button {
+    top: -3.9rem;
+    left: -0.8rem;
+    width: max-content;
+    padding: 0.2rem 0.5rem;
```

Comment:
consider using *px* instead of rem when value is < 1rem or 10px, as told by Vikram Sir